core/src/main/java/cucumber/runtime/formatter/PrettyFormatter.java

```diff
@@ -150,8 +151,17 @@ private void handleTestStepFinished(TestStepFinished event) {
     }
 
     private void handleWrite(WriteEvent event) {
-        out.println(event.text);
-    }
+        out.println();
+        try(BufferedReader lines = new BufferedReader(new StringReader(event.text))) {
+		String line;
```

Comment:
Looks better. But still not quite right. If you are using an IDE there usually is a short cut to do this for you.